.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: Continuous Integration
```

Comment:
more fitting here would be "delivery", because the workflow is about building a runnable version of the tool (see https://www.atlassian.com/continuous-delivery/principles/continuous-integration-vs-delivery-vs-deployment). 

Because we'll see this name on the check summary page, I would be more descriptive and say something like "build and push to dockerhub" or something similar